posture/serialnumber_notmacos.go

```diff
@@ -71,7 +70,6 @@ func init() {
 		validTables = append(validTables, table)
 	}
 	numOfTables = len(validTables)
-
 }
 
 // serialFromSmbiosStructure extracts a serial number from a product,
```

Comment:
This is doing the same checking of `s.Header.Type` as the `switch` statement on line 111, so perhaps we can just call `getStringFromSmbiosStructure` from the loop below, removing the need for `serialFromSmbiosStructure`?

Reply:
Done, a lot neater.